static/js/biomedical/landing/highlights_section.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Section calling out Biomedical KG stats on the Biomedical DC landing page.
+ */
+
+import React from "react";
+import { styled } from "styled-components";
+
+import { BREAKPOINTS, ContentContainer } from "./shared";
+
+const HighlightsContainer = styled(ContentContainer)`
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 32px;
+  justify-content: space-evenly;
+  text-align: center;
+`;
+
+const Label = styled.div`
+  display: flex;
+  flex-direction: column;
+  font-size: 22px;
+  font-weight: 400;
+  gap: 16px;
+  line-height: 28px;
+
+  @media ${BREAKPOINTS.md} {
+    font-size: 16px;
+    line-height: 20px;
+  }
+`;
+
+const Highlight = styled.div`
+  color: ${(props) => props.theme.highlightColors.dark};
+  font-size: 45px;
+  font-weight: 400;
+  line-height: 52px;
+
+  @media ${BREAKPOINTS.md} {
+    font-size: 28px;
+    line-height: 36px;
+  }
+`;
+
+interface HighlightsSectionProps {
+  config: Record<string, number>[];
+  locale: string;
+}
+
+export function HighlightsSection(props: HighlightsSectionProps): JSX.Element {
+  return (
+    <HighlightsContainer className="container">
+      {props.config.map((callout, index) => {
+        const label = Object.keys(callout)[0];
+        if (label === "Entities") {
+          const value = Intl.NumberFormat(props.locale, {
```

Comment:
curious if internationalization of the numbers is something we are worried about? If not, we could just update the config to be formatted like: 

```
[
  {
    label: string,
    value: string
  },
  ...
]
```
Actually, even if you do want internationalization of the numbers, this format may be cleaner instead of getting the label by reading the first key of the object

Reply:
jennifer's suggestion sgtm. Seems too tightly coupled to have the entities label in the config file and an override for it here.

Reply:
also depending on the type of `"value"` you could apply number internationalization or not.

```
const displayValue = typeof item.value === 'number' ? formatter.format(item.value) : item.value
```

Reply:
Good ideas. I updated the config to use "label" and "value" keys, and also added logic to only format if the value is a number.